enumfields/fields.py

```diff
@@ -128,11 +128,24 @@ def formfield(self, form_class=None, choices_form_class=None, **kwargs):
 
 
 class EnumField(EnumFieldMixin, models.CharField):
-    def __init__(self, enum, **kwargs):
-        kwargs.setdefault("max_length", 10)
+    def __init__(self, enum, auto_length=False, **kwargs):
+        max_item_length = self._get_max_item_length(enum)
+        kwargs.setdefault("max_length", (max_item_length // 10 + 1) * 10 if auto_length else 10)
+
+        assert max_item_length <= kwargs["max_length"], 'The enum have value longer that max_length of the field!'
```

Comment:
This shouldn't be an `assert` but a real `if`/`raise ValueError`. (`assert`s could be optimized away.)

Also, maybe "Enum has value(s) longer than the declared max_length (%d)" or similar?

Reply:
About `assert`: I think we have to check this during developing and CI tests only but not in the production. It is the way that used by Django REST Framework. (But `raise ValueError` is also OK, so I can change it.)
What do you think?

About message: agree.